src/main/resources/com/mathworks/ci/RunMatlabCommandBuilder/help-matlabCommand.html

```diff
@@ -1,7 +1,8 @@
 <div>
- To run your tests, enter MATLAB commands separated by semicolons.<br>
- <b>Example:</b> cd c:\MATLAB\tests; runtests('IncludingSubfolder','true');<br>
- 
+ Enter the command you want to run in MATLAB. If the command represents a MATLAB function or script, do not specify the file extension.<br>
+ <b>Examples:</b><br>
+ <b>To run MATLAB commands :</b> cd c:\MATLAB\tests; runtests('IncludingSubfolder','true')<br> 
```

Comment:
Commands: 
cd c:\MATLAB\tests; runtests('IncludingSubfolder','true')

Reply:
Done.

Reply:
In line with my comment below, I would rewrite this statement as follows:

<b>Run MATLAB commands :</b> cd C:\MATLAB\tests; runtests('IncludingSubfolder','true')<br>